lib/order.rb

```diff
@@ -0,0 +1,26 @@
+require_relative '../lib/menu'
+
+class Order
+  attr_reader :basket, :menu, :dishes
+
+  def initialize
+    @basket = Hash.new(0)
+    @order_total = 0 
+    @menu = Menu.new
+  end
+
+  def add_to_order(dish, quantity)
```

Comment:
Suggestion: It took me a couple of tries to realise I needed to pass the dish name as a symbol not a string in irb, so this could be something to look at. Maybe consider including a note or example in the README to specify how to order.